dbcat/catalog/models.py

```diff
@@ -174,6 +177,8 @@ def conn_string(self):
                     else "",
                 )
             )
+        elif self.source_type == "oracle":
+            conn_string = f"oracle+cx_oracle://{self.username}:{self.password}@{self.uri}:{self.port}/{self.service_name}"
```

Comment:
I am not familiar with Oracle. Is `service_name` the equivalent of a database within oracle? Postgres has a similar concept.

Reply:
Oracle's Service Name is a way to point to a database (or databases) in the Oracle DB (like DNS), and the Postgres database is the actual database within the Postgres DB.

Reply:
@vjarat I am sorry to push, but we're currently using piicatcher as a stage in our pipeline. It would be great for us to have an Oracle support merged -- if there's anything more i can do to improve this PR?

Reply:
Sorry for the delay. I'll look into this today.